### PR TITLE
Upload Image of the Project Thumbnail/Logo

### PR DESCRIPTION
**Pull request by popkinj**
## Image Upload Using Common Upload Component
- Thumbnail Creation
- Thumbnail addition to the project creation page
- Enhancements to the File Upload component
- Delete button for the thumbnail

Screenshot from 2024-05-30 09-13-02
Screenshot from 2024-05-30 09-14-26
Screenshot from 2024-05-30 09-17-46
Screenshot from 2024-05-30 09-23-08

### CODE REVIEW

**Comment by popkinj** [comment]
I snuck this feature in.... Slight indicator that the file can be successfully dropped.
Untitled

File: app/src/components/attachments/ImageUpload.tsx [hunk 1]
```diff
@@ -37,12 +38,39 @@ const ImageUpload = () => {
...
+        const newHeight = Math.sqrt((res * res) / aspectRatio);
+        canvas.width = newWidth;
+        canvas.height = newHeight;
+
+        ctx?.drawImage(img, 0, 0, newWidth, newHeight);
+
+        const dataUrl = canvas.toDataURL();
+
+        setImage(dataUrl);
+      };
```
Comment (KjartanE) [inline app/src/components/attachments/ImageUpload.tsx]: Just a nit on PR naming. NERT-361 is focused on upload to S3 buckets. I believe NERT-365 is more correct for this PR

**Comment by popkinj** [comment]
Just moved the native styling over to MUI. Think it looks pretty good.
Screenshot from 2024-05-31 11-07-02
Screenshot from 2024-05-31 11-07-19

File: app/src/components/attachments/ImageUpload.tsx (hunk 1 above)
Comment (popkinj) [reply app/src/components/attachments/ImageUpload.tsx]: Good point. Just renamed it.

**Review by oscar-ip — APPROVED** [review]
looks great!! :-)

**Pull request merged**